src/Common/src/Interop/Windows/Ole32/Interop.IStream.cs

```diff
@@ -10,7 +10,7 @@ internal static partial class Interop
     internal static partial class Ole32
     {
         /// <summary>
-        /// COM IStream interface. <see cref="https://docs.microsoft.com/en-us/windows/desktop/api/objidl/nn-objidl-istream"/>
+        /// COM IStream interface. See https://docs.microsoft.com/en-us/windows/desktop/api/objidl/nn-objidl-istream.
```

Comment:
❔ Can we use `<see href="uri"/>` here so the link is preserved as a link in the doc metadata? Also applies to a few other cases.

Reply:
Sure